src/production/stages/codeSplittingStage.ts

```diff
@@ -45,9 +45,12 @@ export function codeSplittingStage(props: IProductionFlow) {
     processEntry(props, link);
 
     if (root.dynamicDependencies.length > 0) {
+      const useHash = props.ctx.config.codeSplitting.useHash
```

Comment:
could you make it `true` if it's undefined?

Reply:
done